src/sentry/api/endpoints/organization_details.py

```diff
@@ -186,6 +187,12 @@ def validate_slug(self, value):
         qs = Organization.objects.filter(slug=value).exclude(id=self.context["organization"].id)
         if qs.exists():
             raise serializers.ValidationError(f'The slug "{value}" is already in use.')
+
+        contains_whitespace = any(c in self.initial_data["slug"] for c in string.whitespace)
```

Comment:
nit: you could declare a set of of the whitespaces above `set(string.whitespace)` and iterate through `self.initial_data["slug"]` just once: `any(c in whitespace_set for c in self.initial_data["slug"])`

Reply:
@JoshFerge I went with `any(c.isspace() for c in self.initial_data["slug"])`

Reply:
nice 👍🏼